src/Chainweb/Version.hs

```diff
@@ -670,7 +671,8 @@ enableModuleNameFix
     -> BlockHeight
     -> Bool
 enableModuleNameFix v bh = case v of
-  Mainnet01 -> forHeight 500_000 -- TODO
+  Mainnet01 -> forHeight 448501 -- ~ 2020-04-02T12:00:00Z
```

Comment:
@fosskers please confirm this one as well.

Reply:
@fosskers could you also confirm the testnet dates, that I put in after your first review?

Reply:
my calculations have 
- 4/2 12:01 PM - mainnet 👍 
- 4/2 12:56 AM - testnet 👍